.eslintrc

```diff
@@ -16,6 +15,19 @@
     "no-param-reassign": "off",
     "consistent-return": "off",
     "vars-on-top": "off",
-    "compat/compat": "error"
+    "compat/compat": "error",
```

Comment:
because import/export (es6 features) weren't supported by our super old eslint rules, I had to update them and disable everything that was against those rules.